src/SonarScanner.MSBuild.Shim/Resources.Designer.cs

```diff
@@ -84,10 +83,10 @@ internal class Resources {
         /// <summary>
         ///   Looks up a localized string similar to The SonarScanner for MSBuild integration failed: {0} was unable to collect the required information about your projects.
         ///Possible causes:
-        ///  1. The project has not been built - the project must be built in between the begin and end steps
-        ///  2. An unsupported version of MSBuild has been used to build the project. Currently MSBuild 14.0.25420.1 and higher are supported.
-        ///  3. The begin, build and end steps have not all been launched from the same folder
-        ///  4. None of the analyzed projects have a valid Projec [rest of string was truncated]&quot;;.
+        ///  1. The project has not been built - the project must be built in between the begin and end steps.
+        ///  2. An unsupported version of MSBuild has been used to build the project. Supported versions: MSBuild 16 and higher.
+        ///  3. The begin, build and end steps have not all been launched from the same folder.
+        ///  4. None of the analyzed projects have a valid ProjectGuid and yo [rest of string was truncated]&quot;;.
```

Comment:
Educational: is "yo" a typo? If not what does it stand for?

Reply:
Oh I guess it should be "and you have not used a solution (.sln)."

Reply:
This is the designer file, and the content gets trimmed. See the rest of the string: 
> [rest of string was truncated]&quot;;.

The same behavior was also present in the previous version of the code with a different location.